players/base.py

```diff
@@ -0,0 +1,197 @@
+import memory
+from typing import List
+import xbox
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class Player():
+    def __init__(self, name: str, id: int, battle_menu: List[int]):
+        self.name = name
+        self.id = id
+        self.struct_offset = id * 94
```

Comment:
This is decimal rather than hexadecimal, is that intentional? I don't know how the memory map looks.

Reply:
It wasn't.